Formula/qscintilla2.rb

```diff
@@ -18,89 +17,80 @@ class Qscintilla2 < Formula
     sha256 cellar: :any, mojave:        "20794671c986947c27acddf7d5770559ac1567a445b452a99af2669089493cc5"
   end
 
-  depends_on "pyqt"
+  depends_on "pyqt-builder" => :build
+
+  # TODO: use qt when octave can migrate to qt6
+  depends_on "pyqt@5"
   depends_on "python@3.9"
   depends_on "qt@5"
   depends_on "sip"
 
-  # Fix for rpath in library install name. Taken from
-  # https://github.com/macports/macports-ports/pull/7790
-  # https://www.riverbankcomputing.com/pipermail/qscintilla/2020-March/001444.html
-  patch :DATA
-
   def install
+    # TODO: when qt 6.1 is released, modify the spec
     spec = (ENV.compiler == :clang) ? "macx-clang" : "macx-g++"
     spec << "-arm64" if Hardware::CPU.arm?
     args = %W[-config release -spec #{spec}]
 
-    cd "Qt4Qt5" do
+    pyqt = Formula["pyqt@5"]
+    qt = Formula["qt@5"]
+    xy = Language::Python.major_minor_version Formula["python@3.9"].opt_bin/"python3"
+
+    cd "src" do
       inreplace "qscintilla.pro" do |s|
+        s.gsub! "QMAKE_POST_LINK += install_name_tool -id @rpath/$(TARGET1) $(TARGET)",
+          "QMAKE_POST_LINK += install_name_tool -id #{lib}/$(TARGET1) $(TARGET)"
```

Comment:
```suggestion
          "QMAKE_POST_LINK += install_name_tool -id #{opt_lib}/$(TARGET1) $(TARGET)"
```
Can we use a more stable path here?

Reply:
`opt_lib` is not existed during installation, but `brew` still fix the identification name when finishing installation.